src/reports.coffee

```diff
@@ -196,8 +205,14 @@ setupAgenda = (agenda) ->
   agenda.define 'send daily channel metrics', (job, done) ->
     sendReports job, 'dailyReport', done
 
-  agenda.every "1 weeks", 'send weekly channel metrics'
-  agenda.every "1 days", 'send daily channel metrics'
+  agenda.every config.reports.weeklyReportAt, 'send weekly channel metrics'
+  agenda.every config.reports.dailyReportAt, 'send daily channel metrics'
+
+  agenda.on "start", (job)->
+    logger.info "starting job: " + job.attrs.name
+
+  agenda.on "fail", (err, job)->
+    logger.error "Job " + job.attrs.name + " failed with " + err.message
```

Comment:
Perhaps add an event listener for 'complete' tasks as well.

Also, perhaps we should setup these event [here rather](https://github.com/jembi/openhim-core-js/blob/master/src/server.coffee#L202-L204) as places other than reports use agenda, so it will make it a bit clearer that jobs will be logged.